docs/setup_docs.sh

```diff
@@ -54,6 +58,7 @@ cd tmp
 integrate_connector_docs elasticsearch v3.0.0
 integrate_connector_docs aws v3.0.0
 integrate_connector_docs cassandra v3.0.0
+integrate_connector_docs pulsar main
```

Comment:
No, because the externalized Pulsar connector in `main` is only compatible with Flink 1.17 (because of FLINK-28853) so it can only be updated when Flink 1.17 is released and there has been a new Pulsar release created (v4.0.0)

Reply:
I see, thanks for clarification.